wms/models/res_config_settings.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Camptocamp (https://www.camptocamp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    module_stock_vertical_lift_kardex = fields.Boolean(
```

Comment:
@guewen I don't really like this way of doing settings. IMHO each entry would be added in each module.

Reply:
But the entry is for installing the module, then it wouldn't be there until you install it?

Reply:
Did you check in 12.0? I did this by mistake, forgot to `pip install -e` a module :) and I had no stack trace, nothing happened, the checkbox reverts to a blank box on save. A bit confusing, but no stack trace at least ;)

Reply:
Ok, good to know. It occured in past ones.